manifests/operators/prometheus-remote-bosh-exporter.yml

```diff
@@ -0,0 +1,21 @@
+# Depends on ./prometheus-bosh-exporter.yml
```

Comment:
I'm using it with bucc: https://github.com/starkandwayne/bucc/pull/20. If rolling out clean envs there is a circular dependency between the bosh tsdb health monitor endpoint and the prometheus deployment which you would deploy with said bosh. The solution is colocating the boshtsdb-exporter with bosh during initial deploy. And let prometheus scrap it once its deployed.